package.json

```diff
@@ -49,6 +49,7 @@
     "tippy.js": "^6.3.7",
     "uuid": "^9.0.0",
     "v-tooltip": "2.1.3",
+    "vue": "3.1.5",
```

Comment:
because its a peer that need to be shipped by the leading project.
But, true, It doesn't have to be moved. I think in the End it shouldn't make any difference. I will role it back.

Reply:
f76ad9ab92522c740309aac4354fdb5ed6aa20fd